src/visualize/components/layout.py

```
@@ -0,0 +1,953 @@
+    # 言語フィルターのオプションを作成
+    language_options = [{"label": "All Languages", "value": "all"}]
+    language_options.extend(
+        [{"label": lang, "value": lang} for lang in available_languages]
+    )
+
+    # 既存の散布図ビューのコンテンツ
+    # プロジェクト選択はタブの外に出すため、ここではフィルタから開始
+    scatter_view_content = html.Div(
+        className="container",
+        children=[
+            # 上部カード：コントロールパネルとプロジェクト概要
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)